### PR TITLE
Integration Testing Pipe

### PR DESCRIPTION
**Pull request by gowrizrh**
I've committed to the main branch in order to have the image built and tested as a starting point. Now that I'm done this PR is into a dummy branch purely only for code review.

This likely is my first bash scripting attempt. Feel free to suggest even the slightest improvement :memo:

### CODE REVIEW

File: Dockerfile
Comment (gowrizrh) [inline Dockerfile:1]: note: please expand and review this entire Dockerfile

File: README.md
Comment (aligent-lturner) [inline README.md]: Can give an example for commerce here too?

File: README.md
Comment (aligent-lturner) [inline README.md]: Is this two groups or one? Not completely clear from an instructional point-of-view. Could potentially also reference the `@group` annotation

File: README.md
Comment (aligent-lturner) [inline README.md:31]: Can one of the examples below use the `GROUP` variable?

File: README.md
Comment (gowrizrh) [reply README.md]: I've added this

File: README.md
Comment (gowrizrh) [reply README.md]: This should be updated now

File: README.md
Comment (gowrizrh) [reply README.md:31]: Added an example for `GROUP` var usage

**Comment by aligent-lturner** [comment]
@gowrizrh Looks ok to me, but I'd wait for a review from devops.
Might be worth adding a CODEOWNERS file and requiring owner approval for PRs

File: pipe.sh
Comment (daniel-vdp) [inline pipe.sh:51]: These 6 or so lines are duplicated a few times. Can we pull this into a `composer_config_setup()` function or something similar?

**Review by daniel-vdp — CHANGES_REQUESTED** [review]
Minor comment but looks good to me :tada:

File: pipe.sh
Comment (aligent-lturner) [reply pipe.sh:51]: Yep - the latest commit showed the need for this. Extra lines were added only to the graphql function and not the other 2

Comment (gowrizrh) [reply pipe.sh:51]: thanks, i'll resolve this next

Comment (gowrizrh) [reply pipe.sh:51]: this is resolved by #2

**Review by daniel-vdp — APPROVED** [review]
LGTM

**Pull request merged**
